src/controllers/taskUpdatesHandler.ts

```diff
@@ -0,0 +1,28 @@
+import { env } from "../typeDefinitions/default.types";
+import JSONResponse from "../utils/JsonResponse";
+import * as response from "../constants/responses";
+import { verifyNodejsBackendAuthToken } from "../utils/verifyAuthToken";
+import { sendTaskUpdate } from "../utils/sendTaskUpdates";
+
+export const sendTaskUpdatesHandler = async (request: any, env: env) => {
+  try {
+    const authHeader = request.headers.get("Authorization");
+    if (!authHeader) {
+      return new JSONResponse(response.BAD_SIGNATURE);
+    }
+    await verifyNodejsBackendAuthToken(authHeader, env);
+    const updates: {
+      content: { completed: string; planned: string; blockers: string };
+    } = await request.json();
```

Comment:
- create a type for this above the `sendTaskUpdatesHandler` as it will allow us to reuse this type in different methods if required.

Reply:
Done.